### DISCUSSION TITLE
Using semantic-release on Jenkins says it was triggered on origin/main, while semantic-release is configured to main

### DISCUSSION THREAD

**Discussion by jaidevmane** [semantic-release/semantic-release#1822] [Q&A]
[semantic-release] › ℹ  This test run was triggered on the branch origin/main, while semantic-release is configured to only publish from main, therefore a new version won’t be published.

I have configured branches to be 'main' in the package.json, and am using a webhook on Github

**Comment by jaidevmane** [comment]
Update: The issue above has been resolved

**Reply by travi** [reply]
Mind sharing your solution in case others run into a similar situation?
